redfishtool/Chassis.py

```diff
@@ -745,6 +748,67 @@ def clearLog(self,sc,op,rft,cmdTop=False, prop=None):
         rft.printVerbose(4,"{}:{}: in operation".format(rft.subcommand,sc.operation))
         return(8,None,False,None)
 
+    
+    def getSensors(self,sc,op, rft, cmdTop=False, prop=None):
+        rft.printVerbose(4,"{}:{}: in operation".format(rft.subcommand,sc.operation))
+
+        sys.stdout.write("%-50s %-10s %-10s\n" % ("Sensor", "Reading", "UpperCritical"))
+        resName="Thermal"
+        # get the Chassis resource first
+        rc,r,j,d=op.get(sc, op, rft, cmdTop, resName)
+        if( rc != 0):  return(rc,r,False,None)
+
+        # get the link to the Thermal resource under Chassis
+        if ((resName in d) and ("@odata.id" in d[resName])):
+            resLink=d[resName]["@odata.id"]
+        else:
+            rft.printErr("Error: Chassis resource does not have a {} link".format(resName))
+            return(6,None,False,None)
+
+        if cmdTop is True:   prop=rft.prop
+
+        rc,r,j,d=rft.rftSendRecvRequest(rft.AUTHENTICATED_API, 'GET', r.url, relPath=resLink, prop=prop)
+        if(rc==0):
+            rft.printVerbose(1," {} Resource ".format(resName,skip1=True, printV12=cmdTop))
+
+        collection = ['Temperatures','Fans']
+        for collName in collection:
+          numOfLinks=len(d[collName])
+          for i in range (0,numOfLinks):
+            if collName== 'Fans':
+                sys.stdout.write("%-50s %-10s %-10s\n" % (d[collName][i]["MemberId"], d[collName][i].get("Reading",{}),"N/A"))
```

Comment:
Small correction: If "Reading" is missing, show "N/A" rather than empty dict.

```suggestion
                sys.stdout.write("%-50s %-10s %-10s\n" % (d[collName][i]["MemberId"], d[collName][i].get("Reading","N/A"),"N/A"))
```